volontulo/tests/views/test_offers.py

```diff
@@ -137,3 +137,96 @@ def test_offers_list_for_admin(self):
+            user=organization_user,
+        )
+        cls.organization.save()
+        # pylint: disable=no-member
+        cls.organization.organizations.add(organization)
+
+    def setUp(self):
+        u"""Set up each test."""
+        self.client = Client()
+
+    def test_offers_create_get_method(self):
+        u"""Test page for offer creation - tendering template with form."""
+        self.client.post('/login', {
+            'email': u'organization@example.com',
+            'password': '123org',
+        })
+        response = self.client.get('/offers/create')
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'offers/offer_form.html')
+
+    def test_offers_create_invalid_form(self):
+        u"""Test attempt of creation of new offer with invalid form."""
+        self.client.post('/login', {
+            'email': u'organization@example.com',
+            'password': '123org',
+        })
+        response = self.client.post('/offers/create', {})
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'offers/offer_form.html')
+        self.assertContains(
+            response,
+            u'Formularz zawiera niepoprawnie wypełnione pola'
+        )
+
+    def test_offers_create_valid_form(self):
+        u"""Test attempt of creation of new offer with valid form."""
+        self.client.post('/login', {
+            'email': u'organization@example.com',
+            'password': '123org',
+        })
+        for i in range(1, 10):
+            response = self.client.post('/offers/create', {
+                'organization': u'1',
+                'description': u'required description',
+                'requirements': u'required requirements',
+                'time_commitment': u'required time_commitment',
+                'benefits': u'required benefits',
+                'location': u'required location',
+                'title': u'volontulo offer',
+                'time_period': u'required time_period',
+            }, follow=True)
+            self.assertRedirects(
+                response,
+                '/offers/volontulo-offer/{}'.format(i),
+                302,
+                200,
+            )
+            offer = Offer.objects.get(id=i)
+            self.assertEqual(
+                offer.organization,
+                self.organization.organizations.all()[0],
+            )
+            self.assertEqual(offer.description, u'required description')
+            self.assertEqual(offer.requirements, u'required requirements')
+            self.assertEqual(
+                offer.time_commitment,
+                u'required time_commitment'
+            )
+            self.assertEqual(offer.benefits, u'required benefits')
+            self.assertEqual(offer.location, u'required location')
+            self.assertEqual(offer.title, u'volontulo offer')
+            self.assertEqual(offer.time_period, u'required time_period')
```

Comment:
I think to be more precise there should be also checking for error responses - information about fields requirements (error messages) after form submitting. (I'd like to talk about this)